state-chain/pallets/cf-broadcast/Cargo.toml

```
@@ -22,6 +22,7 @@ cf-primitives = { workspace = true }
 cf-traits = { workspace = true }
 cfe-events = { workspace = true }
 cf-runtime-utilities = { workspace = true }
+generic-typeinfo-derive = { path = '../../generic-typeinfo-derive'}
```

Please put it in the workspace toml :)

```suggestion
generic-typeinfo-derive = { workspace = true }
```